scripts/travis-install

```diff
@@ -15,10 +15,20 @@ then
     sudo apt-get autoremove
     sudo apt-get autoclean
     sudo rm -rf /var/lib/mysql
-    sudo truncate -s 0 /var/log/mysql/error.log
 
+    # Install Elasticsearch
+    wget https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.7.2.deb
+    sudo dpkg -i elasticsearch-1.7.2.deb
+    
```

Comment:
I don't think it is necessary to install a new ElasticSearch service, instead of using the one provided by TravisCI.  This would only be used by the existing py27 tests, and they were happy with the TravisCI version.